addons/hr_holidays/static/src/scss/time_off.scss

```diff
@@ -46,5 +46,11 @@
         .o_leave_stats {
             margin-bottom: 16px;
         }
+        .o_time_off_attachments {
+            .o_attachment {
+                width: 350px;
+                padding-left: 0;
+            }
+        }
```

Comment:
While this does fix this specific issue, it seems to me that this is a wider Owl issue that should probably be addressed everywhere, not just in time off. For example I see the same issue in the appraisals app, when clicking on the "ask feedback" wizard on any given appraisal.

Reply:
Hello @robcoekaerts 

The issue is there only if the parent has less width than the component itself(Like col-6)
Yes, it would be better to fix the issue in the web itself